src/state/migrations/index.ts

```diff
@@ -95,6 +96,7 @@ export const clearAssetsMigrations = {
   61: clearAssets,
   62: clearAssets,
   63: clearAssets,
+  64: clearAssets,
```

Comment:
local wallet migration makes sense with the key update, but do we need this?

Reply:
We *should* not, but might as well? `name` changed in the slice so just to be safe...

Reply:
oh right, was just looking at the top level reducers object initially. sounds good!